app/controllers/application_controller.rb

```diff
@@ -2262,4 +2262,16 @@ def setup_live_events_context
   def teardown_live_events_context
     LiveEvents.clear_context!
   end
+
+  def launch_darkly_user
+    domain = ENV['SETTINGS_TABLE_PREFIX'] || 'localhost'
```

Comment:
is `|| 'localhost'` necessary here? I think the `SETTINGS_TABLE_PREFIX` was only set to `'settings'` on an out of date version of the `canvas.env` file? I may be wrong on that though...

Reply:
Also, why are we adding this to the `ApplicationController` in LMS and the shim?

Reply:
shim shouldn't have any changes for this feature. will remove.

Reply:
yea - it looks like the SETTINGS_TABLE_PREFIX is required so we don't need to have the default localhost here.